homeassistant/components/ezviz/sensor.py

```diff
@@ -18,25 +18,64 @@
 PARALLEL_UPDATES = 1
 
 SENSOR_TYPES: dict[str, SensorEntityDescription] = {
-    "sw_version": SensorEntityDescription(key="sw_version"),
     "battery_level": SensorEntityDescription(
         key="battery_level",
+        name="Battery",
+        translation_key="battery_level",
```

Comment:
In this case, we can even omit the translation key. In that case, Home Assistant will automatically fallback onto the translation provided by the device class; in this case "Battery".